app/src/main/java/com/amaze/filemanager/application/AppConfig.java

```diff
@@ -68,7 +68,7 @@
     reportSenderFactoryClasses = AcraReportSenderFactory.class)
 public class AppConfig extends GlideApplication {
 
-  private static final Logger LOG = LoggerFactory.getLogger(AppConfig.class);
+  private Logger log = null;
```

Comment:
What happened here?

Reply:
This needs to be initialized at application startup lifecycle instead of static initialization for logfile to be initialized.

Reply:
Why?